tool/gravity/cli/clusterconfig.go

```diff
@@ -219,6 +219,7 @@ func (r configInitializer) newOperationPlan(
 	localEnv, updateEnv *localenv.LocalEnvironment,
 	clusterEnv *localenv.ClusterEnvironment,
 	leader *storage.Server,
+	userConfig interface{},
```

Comment:
nit: If parameter is unused, can we use blank identifiers instead?

Reply:
I'm matching the function signature to the interface for newOperationPlan